relay/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
```

Comment:
Maybe let's create a `rust_apps` folder in the repo root similar to how we have `apps` for Elixir now? Later, I will rename `apps` to `elixir_apps` to make everything clean. This potentially gives us an easier time with `.CODEOWNERS` file and slightly simplifies code-review rules per language we are using.

Reply:
Sounds good!

I'll set things up as a workspace then.

Reply:
Or giving a second thought, we can have `rust`/`elixir`/etc folders at the root and I'll not just rename apps but move all elixir guts (like `mix.exs`, `rel`, `config` folders) to the `elixir/` completely so that rust devs won't be annoyed by a bunch of elixir files :).

WDYT @conectado?

Reply:
Yeah sounds good.

Also having a workspace for all rust crates is a good idea